.github/workflows/maven-old-tp.yml

```diff
@@ -0,0 +1,54 @@
+# It builds using an old Eclipse TP
+
+name: Java CI with Maven Old TP
+
+on:
+  push:
+    paths-ignore:
+     - 'README.md'
+  pull_request:
+    paths-ignore:
+     - 'README.md'
```

Comment:
Maybe ignore all markdowns (`**.MD`) to prevent builds when modifying the contribution guide and the changelog?

Reply:
right! I'll do that!